### ISSUE TITLE
fix a11y issues in org documentation/html/table 

### ISSUE THREAD

**Issue by lareplog** [patternfly/patternfly-next#2165]
for https://github.com/patternfly/patternfly-org/issues/1319 , need to address two a11y issues:

1. scope attribute should be used correctly 
2. Checkbox inputs with the same name attribute value must be part of a group
